app/Http/Controllers/BankAccountsController.php

```diff
@@ -15,7 +16,15 @@ class AccountsController extends Controller
     public function index()
     {
         //
-        return view('banking.accounts.index');
+        $bank_accounts = BankAccounts::all();
+        $coa_number = 1030;
+        // $coa_last_record = BankAccounts::orderBy('created_at', 'desc')->first();
+        // if (empty($coa_last_record)) {
+        //     $coa_number = 1030;
+        // } else {
+        //     $coa_number = $coa_last_record->chartOfAccount->chart_of_account_no + 1;
+        // }
```

Comment:
Allow me to clarify this part first either to strictly set its COA number accordingly to `103X+` or can the user specify by him/herself since its possible to do so in COA itself.